scripter/scripter.go

```diff
@@ -80,14 +100,14 @@ type ScrAbTester interface {
 }
 
 //WithConfig returns a function to attach the config to the scripter
-func WithConfig(c toml.Primitive) func(Scripter) error {
+func WithConfig(c toml.Primitive) ScripterFunc {
 	return func(scr Scripter) error {
 		return toml.PrimitiveDecode(c, scr)
 	}
 }
 
 //SetBasicMethods sets methods that can be called by each script, returning basic functionality
-func SetBasicMethods(c ScrConn, service string) {
+func SetBasicMethods(s Scripter, c ScrConn, service string) {
```

Comment:
I think we have to move this function to another file, like `basicFunctions.go` in this package.